test/e2e/serial/tests/non_regression.go

```diff
@@ -191,5 +195,123 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 				return ok
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", targetedNodeName)
 		})
+
+		It("[test_id:48964][tier2] should be able to schedule a guaranteed deployment pod in selective way", func() {
```

Comment:
can we please rephrase what "selective way" means?

Reply:
Sure thing, will do now.